pmm/vcpkg.cmake

```diff
@@ -145,10 +145,26 @@ function(_pmm_vcpkg_default_triplet out)
     set(${out} "${arch}-${platform}" PARENT_SCOPE)
 endfunction()
 
+function(_pmm_vcpkg_copy_custom_ports portsList)
+    foreach (port IN LISTS portsList)
+        get_filename_component(portName ${port} NAME)
+        if (NOT EXISTS "${port}/portfile.cmake")
+            message(FATAL_ERROR "Failed find portfile in ${port}!")
+        else ()
+            if (NOT EXISTS "${vcpkg_inst_dir}/ports/${portName}/portfile.cmake")
```

Comment:
It is a desirable feature to be able to overwrite old ports. I often make changes to some upstream port and want to use it in the ~week or so before it gets merged. I would agree that a soft warning (not an actual cmake warning, but just a note that it is being overwritten) would be good. Alternatively it could refuse to overwrite unless another flag is handed to pmm to enable that feature.

Reply:
I've implemented your feedback, now it compares the timestamps of the files and copies if necessary and gives a warning if the port is already included by default